packages/webpack-cli/lib/utils/cli-executer.js

```diff
@@ -54,7 +54,7 @@ async function prompter() {
 async function run() {
     try {
         const args = await prompter();
-        process.stdout.write('\n');
+        logger.raw('\n');
```

Comment:
I think it can create extra newline, because `console.log` always insert newline after expression, but maybe it is expected here

Reply:
Same for all `logger.raw('\n');`